src/main/flight/pid_init.c

```diff
@@ -423,6 +423,8 @@ void pidInitConfig(const pidProfile_t *pidProfile)
 #endif
 
     pidRuntime.levelRaceMode = pidProfile->level_race_mode;
+    pidRuntime.tpaBreakpoint = (pidProfile->tpa_breakpoint - 1000) * 0.001f; // .35 from 1350, range can be 0 to 0.99, limited by CLI range
```

Comment:
BT uses relaxed math mode, so you can use `/ 1000.0f` when it is more readable. Generated code is equivalent.

Reply:
With relaxed math, compiler is allowed to change order of operations and replace division with multiplication by inverse (and some more). 
Power of two is irrelevant for floating point arithmetic (at least with FPU)

Reply:
Thanks, I finally googled my way through -ffast-math to -freciprocal-math, and yes it looks like it should do the conversion of the division to a multiplication, so long as we keep that option.
Even so I personally find that, at least for division in multiples of 10, a multiplication by an inverse isn't that much harder to follow than a division.